pkg/userop/testing/local_blockchain/helpers.go

```diff
@@ -0,0 +1,65 @@
+package local_blockchain
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+func SendNative(ctx context.Context, node *EthNode, from, to Account, fundAmount *big.Int) error {
+	chainID, err := node.Client.ChainID(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get chain ID: %w", err)
+	}
+
+	nonce, err := node.Client.PendingNonceAt(ctx, from.Address)
+	if err != nil {
+		return fmt.Errorf("failed to get nonce: %w", err)
+	}
+
+	gasLimit := uint64(21000)
+	gasPrice, err := node.Client.SuggestGasPrice(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to suggest gas price: %w", err)
+	}
+
+	tx := types.NewTransaction(nonce, to.Address, fundAmount, gasLimit, gasPrice, nil)
+	signedTx, err := types.SignTx(tx, types.NewEIP155Signer(chainID), from.PrivateKey)
+	if err != nil {
+		return fmt.Errorf("failed to sign transaction: %w", err)
+	}
+
+	err = node.Client.SendTransaction(ctx, signedTx)
+	if err != nil {
+		return fmt.Errorf("failed to send transaction: %w", err)
+	}
+
+	_, err = WaitMined(ctx, node, signedTx)
+	if err != nil {
+		return fmt.Errorf("failed to wait for transaction to be mined: %w", err)
+	}
+
+	return nil
+}
+
+func WaitMined(ctx context.Context, node *EthNode, tx *types.Transaction) (*types.Receipt, error) {
+	queryTicker := time.NewTicker(1 * time.Second)
```

Comment:
Given that these functions are intended for testing, reducing the 1-second delay to a shorter interval, such as 50 milliseconds, would be more appropriate. This adjustment should help expedite the tests without compromising their reliability.

Reply:
03b2806414bea4f017c757d5a577c632ea17f7b9